table/options.go

```diff
@@ -249,3 +250,17 @@ func (m Model) WithColumns(columns []Column) Model {
 
 	return m
 }
+
+// WithFilterInput sets the filter input for the table.  This can be used as API of filter input.
+func (m Model) WithFilterInput(input textinput.Model) Model {
+	m.filterTextInput = input
+
+	return m
+}
+
+// WithFooterVisibility set the visibility of footer.
```

Comment:
```suggestion
// WithFooterVisibility sets the visibility of the footer.
```

Very small grammar fix while we're here.